### PR TITLE
Adapt to new topics and Qt5

### PR DESCRIPTION
**Pull request by christian-rauch**
I implemented some fixes a while ago to test the Schunk hand. It's mainly adapting to new topic names from upstream repo and porting to Qt5 (kinetic and higher).

Although it's very unlikely that we will use the RViz plugin for the Schunk hand again, this will at least allow me to remove my custom branch :-)

### CODE REVIEW

**Comment by wxmerkt** [comment]
Does this work with Kinetic as well or only on Melodic? (Because Qt5)

**Comment by christian-rauch** [comment]
Works with kinetic and melodic. But I think it will break indigo.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,23 +1,26 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sdh_rviz_control_panel)
 
-find_package(catkin REQUIRED)
-find_package(Qt4 REQUIRED COMPONENTS QtCore QtGui)
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```
Comment (wxmerkt) [inline CMakeLists.txt]: Never set `CMAKE_CXX_FLAGS` directly. Use `add_compile_options(-std=c++11)` or another mechanism.

**Review by wxmerkt — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (christian-rauch) [reply CMakeLists.txt]: Right.

**Comment by wxmerkt** [comment]
Thank you for the updates and fixes :)

**Pull request merged**